src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -672,10 +672,10 @@ private void visitEnumType(EnumType type) {
       // var MyEnum: {A: MyEnum, B: MyEnum, ...};
       // </pre>
       // TODO(martinprobst): Special case number enums to map to plain TS enums?
-      visitTypeAlias(type.getElementsType().getPrimitiveType(), getUnqualifiedName(type));
+      String unqualifiedName = getUnqualifiedName( symbolName );
```

Comment:
I keep forgetting I don't have `clang-format` in Java land. We should consider hooking up `google-java-format` at some point.